x-pack/test/functional/config.js

```diff
@@ -82,6 +85,7 @@ export default async function ({ readConfigFile }) {
       resolve(__dirname, './apps/logstash'),
       resolve(__dirname, './apps/grok_debugger'),
       resolve(__dirname, './apps/infra'),
+      resolve(__dirname, './apps/rollupjob'),
```

Comment:
@rasroh I think it is good to follow existing file naming: should be `./apps/rollup_job.js`

Reply:
++ to this suggestion

Reply:
fixed